proto/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -99,7 +100,10 @@ message GetBlockByHeightRequest {
 // GetBlockByHeightResponse is the response type for the Query/GetBlockByHeight RPC method.
 message GetBlockByHeightResponse {
   .tendermint.types.BlockID block_id = 1;
-  .tendermint.types.Block   block    = 2;
+  .tendermint.types.Block   block    = 2 [deprecated = true];
+
+  // since: v0.47
+  Block sdk_block = 3;
```

Comment:
Not sure about the `sdk_block` name. Any other ideas, maybe @alexanderbez ?

Reply:
I guess the main issue with naming here is no matter what in the future we will rename it to block instead of whatever its called. It might be fine for now then